### PR TITLE
Updated the TSSW projects under my purview.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: sal_interfaces/SALSubsystems.xml [hunk 1]
```diff
@@ -759,11 +759,11 @@
...
   <ProductOwner>Brian Stalder</ProductOwner>
   <RelatedDocuments></RelatedDocuments>
-  <SoftwareLanguage></SoftwareLanguage>
+  <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>DATControl</VendorContact>
-  <Simulator>No</Simulator>
-  <Configuration>Yes</Configuration>
+  <Simulator>Internal to CSC</Simulator>
+  <Configuration>Not yet</Configuration>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:766]: This needs to be a guess at a URL. Making one up if there is none is OK for now. If HVAC is part of the EAS, I would use that GH URL.

File: sal_interfaces/SALSubsystems.xml [hunk 2]
```diff
@@ -587,8 +587,8 @@
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Not Applicable</VendorContact>
-  <Simulator>No</Simulator>
-  <Configuration>Not Configurable</Configuration>
+  <Simulator>Internal to CSC</Simulator>
+  <Configuration>Not Yet Configurable</Configuration>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:591]: This needs to be a guess at a URL. Making one up if there is none is OK for now. If ESS is part of the EAS, I would use that GH URL.

File: sal_interfaces/SALSubsystems.xml [hunk 3]
```diff
@@ -556,12 +556,12 @@
 <SALSubsystem>
   <Name>EAS</Name>
   <Description>The CSC for the Environmental Awareness System</Description>
-  <IndexEnumeration>first,second</IndexEnumeration>
+  <IndexEnumeration>no</IndexEnumeration>
   <Generics>yes</Generics>
   <ActiveDevelopers>Wouter van Reeven</ActiveDevelopers>
   <Github></Github>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:562]: Isn't there a GH URL for this already?

File: sal_interfaces/SALSubsystems.xml [hunk 4]
```diff
@@ -556,12 +556,12 @@
...
   <JenkinsTestResults></JenkinsTestResults>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:563]: This can't be empty.

File: sal_interfaces/SALSubsystems.xml [hunk 5]
```diff
@@ -556,12 +556,12 @@
...
-  <RubinObsContact></RubinObsContact>
+  <RubinObsContact>Wouter van Reeven</RubinObsContact>
   <CSCDocs></CSCDocs>
   <ProductOwner>Brian Stalder</ProductOwner>
   <RelatedDocuments></RelatedDocuments>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:567]: GH review won't let me put this where I want. EAS Simulator and Configuration attributes need to have the appropriate content.

File: sal_interfaces/SALSubsystems.xml [hunk 6]
```diff
@@ -587,8 +587,8 @@
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Not Applicable</VendorContact>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:589]: This needs to be Not Available.

File: sal_interfaces/SALSubsystems.xml [hunk 7]
```diff
@@ -759,11 +759,11 @@
   <CSCDocs></CSCDocs>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:759]: GH review won't let me put this where I want. HVAC JenkinsTestResults cannot be empty.

File: sal_interfaces/SALSubsystems.xml (hunk 1 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:766]: There is no CSC for this yet, which is why I set it to "Not yet". I changed it to "Not Configurable".

File: sal_interfaces/SALSubsystems.xml (hunk 2 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:591]: The CSC for this is not configurable yet, which is why I set it to "Not yet". I changed it to "Not Configurable".

File: sal_interfaces/SALSubsystems.xml (hunk 3 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:562]: It turns out there is and I wasn't aware of that.

File: sal_interfaces/SALSubsystems.xml (hunk 4 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:563]: I changed it to "Not Available".

File: sal_interfaces/SALSubsystems.xml (hunk 5 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:567]: I gave those fields appropriate values now.

File: sal_interfaces/SALSubsystems.xml (hunk 6 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:589]: Done

File: sal_interfaces/SALSubsystems.xml (hunk 7 above)
Comment (wvreeven) [reply sal_interfaces/SALSubsystems.xml:759]: I changed it to "Not Available".

**Comment by wvreeven** [comment]
Changes pushed. Can you have another look please?

**Comment by mareuter** [comment]
JenkinsResults content needs to be "Not Applicable" if no Jenkins job has been setup. If that changes, a URL can be added at any point.

**Pull request merged**